tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -3789,6 +3789,7 @@ declare namespace ts {
         readonly importModuleSpecifierEnding?: "auto" | "minimal" | "index" | "js";
         readonly allowTextChangesInNewFiles?: boolean;
         readonly providePrefixAndSuffixTextForRename?: boolean;
+        readonly provideRefactorNotApplicableReason?: boolean;
```

Comment:
I feel like I'm missing something obvious, but shouldn't there also be a protocol change for this?